Documentation/Flashing.md

```diff
@@ -11,7 +11,7 @@ Flash according to details below
 
 ### Bleeding edge / latest
 
-For the _latest_ code, you will need to download the zip file from the artefacts page on the build for what you want.

```

Comment:
Artefact is spelt with an `e` in English (not US)

Reply:
Yeah but it is always spelled "artifact" on the GitHub page. In the next paragraph it was already called "Artifacts" so I thought I'd change it here too. :P

I can change it back if you want.

Reply:
Ah fair it should match Github until they decide to support non-us.